system/lib/fetch/emscripten_fetch.cpp

```diff
@@ -42,6 +42,9 @@ extern "C" {
 		}
 		return queue;
 	}
+
+	int32_t _emscripten_fetch_get_response_headers_length(int32_t);
+	int32_t _emscripten_fetch_get_response_headers(int32_t, int32_t, int32_t);
```

Comment:
Let's add names to the parameters to these functions - they are especially important since the meaning of these is different than the public functions: instead of a pointer to fetch, they take an ID to a fetch, so it's good to highlight this difference.

Reply:
Done.